package.json

```diff
@@ -69,17 +69,19 @@
     "eslint-plugin-react": "^7.16.0",
     "eslint-plugin-react-hooks": "^2.2.0",
     "eslint-plugin-react-native": "^3.8.1",
+    "mocked-env": "^1.3.2",
     "nyc": "^14.1.1",
     "test-listen": "^1.1.0"
   },
   "ava": {
     "babel": false,
     "verbose": true,
     "environmentVariables": {
-      "ACKEE_TRACKER": "custom name"
+      "ACKEE_TRACKER": "custom name",
+      "ACCESS_CONTROL_ALLOW_ORIGIN": "https://test-website.com"
```

Comment:
Can you change this to `ACKEE_ALLOW_ORIGIN` or is this one defined by Heroku?

Reply:
I can change this 👍